rust/operator-binary/src/listener_controller.rs

```diff
@@ -75,10 +112,18 @@ pub enum Error {
     NoName,
     #[snafu(display("object has no ListenerClass (.spec.class_name)"))]
     NoListenerClass,
-    #[snafu(display("failed to generate listener's pod selector"))]
+    #[snafu(display("failed to generate Listener's PersistentVolume selector"))]
+    ListenerPvSelector {
+        source: ListenerPersistentVolumeLabelError,
+    },
+    #[snafu(display("failed to generate Listener's Pod selector"))]
```

Comment:
nit: We normally space the error variants out using newlines in between

Reply:
91336aaef6c62cf6efbcecc2ffe18f136d9bb2f0